packages/xs/nbd.2.3.0/opam

```diff
@@ -11,18 +11,17 @@ build-test: ["jbuilder" "runtest" "-p" name]
 depends: [
   "jbuilder" {build}
   "ounit" {test}
-  "ppx_tools" {build}
   "lwt" {>= "2.6.0" & < "3.0.0"}
-  "cstruct" {>= "1.9.0" & < "3.0.0"}
+  "ssl"
+  "cstruct" {>= "2.4.0" & < "3.0.0"}
```

Comment:
Changing this to `>= "2.3.0"` should be enough to fix the build

Reply:
Testing with this Travis build: https://travis-ci.org/gaborigloi/nbd/builds/272124111